notebooks/tangent_pca_s2.ipynb

```diff
@@ -10,19 +10,18 @@
     "Then it performs tangent PCA at the mean."
```

Comment:
<p>We can end this line by ";" to avoid matplotlib ugly prints (matplotlib.axes... etc), here and later.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>